sdk/net/http/contenttype_go1.14.go

```diff
@@ -0,0 +1,39 @@
+// +build go1.14
+// http.Header.Values is available only in Go 1.14+. This API provides
+// full access to the list of headers.
+
+package http
+
+import (
+	"net/http"
+	"strings"
+)
+
+// shouldRecordBodyOfContentType checks if the body is meant
+// to be recorded based on the content-type and the fact that body is
+// not streamed.
+func shouldRecordBodyOfContentType(h http.Header) bool {
+	for _, contentTypeAllowed := range contentTypeAllowList {
+		for _, contentType := range h.Values("content-type") {
```

Comment:
In go 1.14 we can access all the values fro a header, hence we iterate over them and look for the desired content-type.